### PR TITLE
Added --no-fixtures option to test baking

### PR DESCRIPTION
**Pull request by makallio85**
Based on this one https://github.com/cakephp/bake/pull/227

### CODE REVIEW

**Comment by markstory** [comment]
Looks good. I just want to get at least one passing build in the travis-ci matrix before merging.

File: src/Shell/Task/TestTask.php [hunk 1]
```diff
@@ -215,13 +215,15 @@ public function bake($type, $className)
...
         $fullClassName = $this->getRealClassName($type, $className);
 
-        if (!empty($this->params['fixtures'])) {
-            $fixtures = array_map('trim', explode(',', $this->params['fixtures']));
-            $this->_fixtures = array_filter($fixtures);
-        } elseif ($this->typeCanDetectFixtures($type) && class_exists($fullClassName)) {
-            $this->out('Bake is detecting possible fixtures...');
-            $testSubject = $this->buildTestSubject($type, $fullClassName);
-            $this->generateFixtureList($testSubject);
+        if (empty($this->params['no-fixtures'])) {
```
Comment (antograssiot) [inline src/Shell/Task/TestTask.php]: please use "no-fixture" instead of of "no-fixtures" asit already exists to keep consistency in the command option.

Comment (makallio85) [inline src/Shell/Task/TestTask.php]: Yep, forgot this for some reason. Now it should be ready to go.

**Comment by antograssiot** [comment]
:+1:

**Comment by lorenzo** [comment]
Nice, thanks!

**Pull request merged**